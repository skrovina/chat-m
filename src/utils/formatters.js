// @flow

import { format } from "date-fns"


export const formatDateTime = (date: Date | string): string => format(date, "D.M.YYYY, H:mm")
