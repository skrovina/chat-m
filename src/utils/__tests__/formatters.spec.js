import { formatDateTime } from "../formatters"

describe("formatDateTime", () => {
    it("should format date time correctly", () => {
        expect(formatDateTime("2018-01-21T15:24:48.345Z")).toMatchSnapshot()
    })
})
