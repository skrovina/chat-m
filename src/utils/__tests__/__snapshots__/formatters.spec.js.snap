// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatDateTime should format date time correctly 1`] = `"21.1.2018, 16:24"`;
